gc/2020/GC-10-06.md

```diff
@@ -0,0 +1,282 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 22nd video call of WebAssembly's Garbage Collection Subgroup
+
+- **Where**: zoom.us
+- **When**: October 6th, 4pm-5pm UTC (October 6th, 9am-10am Pacific Daylight Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Fill out the form here to sign up if
+it's your first time: https://forms.gle/JehrAB4gWbtHjybt9. The meeting is open
+to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Discussion: [Superfluous Casts](https://github.com/WebAssembly/gc/issues/120) (Ross Tate) [30 min]
+    1. Discussion: Shared and Separate GC Heaps (Francis McCabe) [30 min]
+1. Closure
+
+## Meeting Notes
+
+### Introduction of attendees
+
+Thomas Lively
+Ryan Hunt
+Daniel Wirtz
+Luke Wagner
+Zalim Bashorov
+Jakob Kummerow
+Derek Schuff
+Ross Tate
+Manos Koukoutos
+Wouter van Oortmerssen
+Tim Steenvoorden
+Rick Battagline
+Conrad Watt
+Benjamin Titzer
+Keith Miller
+Slava Kuzmich
+Sabine
+Lars Hansen
+Yulia Startsev
+Emanuel Ziegler
+Adam Klein
+Andreas Rossberg
+Daniel Ehrenberg
+Francis McCabe
+
+### Discussion: [Superfluous Casts](https://github.com/WebAssembly/gc/issues/120) (Ross Tate) [30 min]
+
+TODO(RT): insert link to slides
```

Comment:
@RossTate please upload your slides when you get a chance :)

Reply:
I made a pull request into your branch with the slides.